LFGBulletinBoard/Localization.lua

```diff
@@ -64,6 +64,7 @@ GBB.locales = {
 		["CboxTagsCustom"]="Custom",
 		["CboxRemoveRaidSymbols"]="Remove raid symbols like {rt1}",
 		["CboxOrderNewTop"]="Sort new requests above",
+		["CboxHeadersStartFolded"]="Headers start folded",
```

Comment:
Can we rename this to like either "Collapse all dungeon categories" or "Fold all dungeons" or like  something along those lines. I don't think most people will know what `Headers` refers to. It's fine to leave the translations the way they are

Reply:
Sure, how about "Dungeon categories start collapsed"? Should I also change the "Fold all"/"Unfold all" text to "Collapse all"/"Expand all" for consistency?

Reply:
Yea that works. To save you time I would say just change `CboxHeadersStartFolded` to match or simply leave it as is and just change the text. That way it's consistent in the code, just the string value itself calls it something different

Reply:
Done